PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,8 @@
 <!-- Please give this pull request a descriptive title. Pull requests with descriptive titles are more likely to receive reviews. Describe what you changed! A title that only references an issue number is not descriptive. -->
 
+
+<!-- Briefly explain what this pull request changes in each line below, with appropriate heading—'M' for major changes, 'm' for minor changes, 'n' for new features, and 'b' for backwards compatibility breaks and deprecations—and a colon(:), e.g., "M: Added changelogs". Write '[skip changelog]' if changes are trivial or not related to SymPy itself. -->
+## This PR changes
+
```

Comment:
Let's add each heading here, so that people can just fill them out. We might as well use the full words too, like 

````
### Major changes

### Minor changes

### New features

### Backwards compatibility breaks

```

That way people can just fill it out like a template.